src/test/resources/examples/inlinedAggregatedObjects/api.yaml

```diff
@@ -8,31 +8,60 @@ paths: { }
 components:
   schemas:
 
-    Dog:
+    Container:
       type: object
       properties:
-        owner:
-          description: "The registered owner of the dog."
+        aggregationOfOne:
           allOf:
-            - $ref: "#/components/schemas/Person"
-        walker:
-          description: "The person walking the dog."
+            - $ref: "#/components/schemas/SimpleObjOne"
+        aggregationOfMany:
           allOf:
-            - $ref: "#/components/schemas/Person"
-            - $ref: "#/components/schemas/Company"
-
-    Person:
+            - $ref: "#/components/schemas/SimpleObjOne"
+            - $ref: "#/components/schemas/SimpleObjTwo"
+        arrayWithAllOfAggregationOfMany:
```

Comment:
This reproduced the issue described here, where the generated classes were referencing non-existent types
https://github.com/cjbooms/fabrikt/issues/392#issuecomment-2821255513